src/firefly/js/metaConvert/converterUtils.js

```diff
@@ -183,10 +183,17 @@ function makeChartObj(chartInfo, activateParams, titleStr, chartId, tbl_id ) {
     const {chartParamsAry}= chartInfo;
     const {xAxis, yAxis}= chartInfo;
 
+    /*The default FileAnalysis does not set the chartParam.  Only the added chart has this parameter set.
+     Thus, if the chartParamsAry is set, the "table_" is the name given by the added analyzer.  Only
+     when chartParamsAry is not defined and the title starts with "table_", this is the table in the HDU that has
+     no extension name.
+     */
+    const chartTitle= !chartParamsAry && titleStr.startsWith('table_')? '':titleStr;
```

Comment:
This is a little confusing.  I think it would be simpler if you just put it in the `else` a fiew lines below. Then it could just be
```js
const chartTitle= titleStr.startsWith('table_')? '':titleStr;
```
I have a mild concern that checking for `'table_'` is to too special. It make this function have to have special knowledge about the contents of `titleStr`. This precludes us from using it in this case but that is probably not that big of deal.

Effectively the jsdoc for this param is: "ignore titleStr if it starts with `table_`"

Reply:
> This is a little confusing. I think it would be simpler if you just put it in the `else` a fiew lines below. Then it could just be
> 
> ```js
> const chartTitle= titleStr.startsWith('table_')? '':titleStr;
> ```
> 
> I have a mild concern that checking for `'table_'` is to too special. It make this function have to have special knowledge about the contents of `titleStr`. This precludes us from using it in this case but that is probably not that big of deal.
> 
> Effectively the jsdoc for this param is: "ignore titleStr if it starts with `table_`"

I had the same concern as you.  I wanted to minimize the changes.  So checking the string is the least change.  I now introduce an object that has two properties, title and showChartTitle.   The change is still very minimal.

Reply:
OK- it is fine to merge.

Reply:
Thank you @robyww !!